en/firefox_privacy_notice.md

```diff
@@ -33,6 +33,8 @@ You can perform searches directly from several places in Firefox, including the
 
 Mozilla generally receives royalties when you use one of the default or optional search options in the Firefox web browser, including but not limited to Google, Bing, DuckDuckGo, and eBay. 
 
+* __Search categories__: In the US, Firefox sends Mozilla the high-level categories of things you search for (e.g. sports, travel, food, etc.) but not your actual search terms, unless you specifically agree to send us that information. Where this occurs, the search category is not shared with our partners and Mozilla is not able to associate the search category with an individual user once the search results have been served. You can opt-out of search category data collection at any time. [Learn more] (https://support.mozilla.org/kb/share-data-mozilla-help-improve-firefox).  
```

Comment:
```suggestion
* __Search categories__: In the US, Firefox sends Mozilla the high-level categories of things you search for (e.g. sports, travel, food, etc.) but not your actual search terms, unless you specifically agree to send us that information. Where this occurs, the search category is not shared with our partners and Mozilla is not able to associate the search category with an individual user once the search results have been served. You can opt out of search category data collection at any time. [Learn more](https://support.mozilla.org/kb/share-data-mozilla-help-improve-firefox).  
```

Broken link (space in the middle of the syntax), opt-out (noun) -> opt out (verb)